projects/manager/src/app/layers/layer-editor/layer-editor.component.ts

```diff
@@ -59,18 +59,22 @@ export interface LayerForm {
         PermissionsComponent,
     ],
 })
-export class LayerEditorComponent implements OnChanges {
+export class LayerEditorComponent {
     private readonly layersService = inject(LayersService);
     private readonly workflowsService = inject(WorkflowsService);
     private readonly dialog = inject(MatDialog);
     private readonly snackBar = inject(MatSnackBar);
     private readonly config = inject(AppConfig);
 
-    @Input({required: true}) layerListing!: LayerListing;
-    @Input({required: true}) parentCollection!: ProviderLayerCollectionId;
+    readonly _layerListing = input.required<LayerListing>({
+        // eslint-disable-next-line @angular-eslint/no-input-rename
+        alias: 'layerListing',
+    });
+    readonly layerListing = linkedSignal(this._layerListing);
```

Comment:
why is linkedSignal needed?

Reply:
Same as above, we wrote to the input. This is not possible anymore, so I created a signal to write to and update if the input changes.